.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -47,26 +55,40 @@ function generateAndroidVersionCode(npmVersion) {
 }
 
 /**
- * Use the react-native-version package to update the version of a native platform.
+ * Update the Android app version.
  *
- * @param {String} platform – "ios" or "android"
- * @param {String} versionCode – The version code to update the native platform.
+ * @param {String} versionName
+ * @param {String} versionCode
  */
-function updateNativeVersion(platform, versionCode) {
-    if (platform !== 'android' && platform !== 'ios') {
-        console.error('Invalid native platform specified!', platform);
-        core.setFailed();
-        return;
-    }
+function updateAndroidVersion(versionName, versionCode) {
+    console.log('Updating android:', `versionName: ${versionName}`, `versionCode: ${versionCode}`);
+    readFileAsync(BUILD_GRADLE_PATH, {encoding: 'utf8'})
+        .then((content) => {
+            let updatedContent = content.replace(/versionName "([0-9.-]*)"/, `versionName "${versionName}"`);
```

Comment:
Are the " required here? The line below doesn't have them?

Reply:
yes, they're required. It's because the [build.gradle](https://github.com/Expensify/Expensify.cash/blob/4a67962982cbd3e49547f7fa7eac5760ed009952/android/app/build.gradle#L151-L152) file actually has quotes around the version name (treats it as a string) but not around the versionCode (treated as a number).